mono/mini/mini-posix.c

```diff
@@ -393,6 +402,13 @@ mono_runtime_posix_install_handlers (void)
 
 	add_signal_handler (SIGABRT, sigabrt_signal_handler, 0);
 
+	gchar *sigterm_env = g_getenv ("MONO_HANDLE_SIGTERM");
```

Comment:
This should be documented in `man/mono.1`.

Reply:
Use `g_hasenv`